Makefile

```diff
@@ -1,17 +1,34 @@
-test:
-	poetry run tox -p auto
+VENV_NAME?=venv
 
-ci:
-	poetry run pytest --cov=stripe
+venv: $(VENV_NAME)/bin/activate
 
-coveralls:
-	poetry run coveralls
+$(VENV_NAME)/bin/activate: setup.py
+	pip install --upgrade pip virtualenv
+	@test -d $(VENV_NAME) || python -m virtualenv --clear $(VENV_NAME)
+	${VENV_NAME}/bin/python -m pip install -U pip tox
+	${VENV_NAME}/bin/python -m pip install -e .
+	@touch $(VENV_NAME)/bin/activate
 
-fmt:
-	poetry run tox -e fmt
+test: venv
+	@${VENV_NAME}/bin/tox -p auto $(TOX_ARGS)
 
-fmtcheck:
-	poetry run tox -e fmt -- --check --verbose
+ci: venv
+	@${VENV_NAME}/bin/python setup.py test -a "-n 8 --cov=stripe"
 
-lint:
-	poetry run tox -e lint
+coveralls: venv
+	@${VENV_NAME}/bin/pip install --upgrade coveralls
+	@${VENV_NAME}/bin/coveralls
+
+fmt: venv
+	@${VENV_NAME}/bin/tox -e fmt
+
+fmtcheck: venv
+	@${VENV_NAME}/bin/tox -e fmt -- --check --verbose
+
+lint: venv
+	@${VENV_NAME}/bin/tox -e lint
+
+clean:
+	@rm -rf $(VENV_NAME) build/ dist/
+
+.PHONY: venv test ci coveralls fmt fmtcheck lint clean
```

Comment:
I didn't want to revert to the pre-2.0 state where we just installed dependencies in the global Python installation. Now we use a virtualenv, but the Makefile makes it largely transparent (i.e. you can still run `make test`, `make fmt`, etc. and it will initialize the venv if necessary and run the command in the venv).